factory/disabled/txCoordinator.go

```diff
@@ -105,6 +105,11 @@ func (txCoordinator *TxCoordinator) CreateMarshalizedReceipts() ([]byte, error)
 	return nil, nil
 }
 
+// GetCreatedInShardMiniBlocks returns nil as is it disabled
+func (txCoordinator *TxCoordinator) GetCreatedInShardMiniBlocks() []*block.MiniBlock {
```

Comment:
`*intrashard*` (naming, optional)

Reply:
just kept the naming from interimProcessors